imports/plugins/core/inventory/register.js

```diff
@@ -1,8 +1,38 @@
 import Reaction from "/imports/plugins/core/core/server/Reaction";
+import config from "./server/config";
 import startup from "./server/no-meteor/startup";
 import schemas from "./server/no-meteor/schemas";
 import xformCatalogBooleanFilters from "./server/no-meteor/utils/xformCatalogBooleanFilters";
 
+const publishedProductFields = [];
+
+// These require manual publication always
+const publishedProductVariantFields = [
+  "inventoryManagement",
+  "inventoryPolicy"
+];
+
+// Additional fields require manual publication only if they are
+// not auto-published on every variant update.
+if (!config.AUTO_PUBLISH_INVENTORY_FIELDS) {
+  publishedProductFields.push(
+    "inventoryAvailableToSell",
+    "inventoryInStock",
+    "isBackorder",
+    "isLowQuantity",
+    "isSoldOut"
+  );
+
+  publishedProductVariantFields.push(
```

Comment:
@willopez If I am reading the auto-publishing code correctly, then I don't see where it updates those two fields. It auto-publishes the others but not those. So I left them as always needing to be manually published.

I agree it may be best for it to work as you say, all fields, but I would treat that as a separate PR because it's a bigger change and I'm not sure what the ramifications would be.

Reply:
OK, that sounds good. Lets create another issue to follow up on the other fields in the future.